test/utils/index.js

```diff
@@ -136,21 +136,49 @@ export const outputBNs = bn => {
   })
 }
 
-export const createTranch = (policy, attrs, ...callAttrs) => {
+export const createTranche = async (policy, attrs, ...callAttrs) => {
   const {
     numShares = 10,
     pricePerShareAmount = 1,
-    premiums = [],
+    premiumsDiff = [],
   } = attrs
 
-  return policy.createTranch(
+  let premiums = []
+  let policyInfo 
+  policyInfo = await policy.getInfo()
+  const policyInitiationDate = policyInfo.initiationDate_
+
+  for (let i = 0; i < premiumsDiff.length; i += 2) {
+    premiums[i] = policyInitiationDate.add(toBN (premiumsDiff[i]))
+    premiums[i + 1] = toBN (premiumsDiff[i + 1])
+  }
+
+  return policy.createTranche(
     numShares,
     pricePerShareAmount,
     premiums,
     ...callAttrs,
   )
 }
 
+export const adjustTrancheDataDiff = (policyInitiationDate, trancheDiffData) => {
```

Comment:
does not need `export`

Reply:
Also, this logic seems to be duplicated within `createTranche()` - pls refactor

Reply:
done.